go/vt/discovery/topology_watcher.go

```diff
@@ -437,3 +437,54 @@ func (fbs *FilterByShard) isIncluded(tablet *topodatapb.Tablet) bool {
 	}
 	return false
 }
+
+// FilterByKeyspace is a TabletRecorder filter that filters tablets by
+// keyspace
+type FilterByKeyspace struct {
+	tr TabletRecorder
+
+	keyspaces map[string]bool
```

Comment:
nit - wondering if we should call this selectKeyspaces to keep it consistent with the naming in:  https://github.com/vitessio/vitess/pull/4420/files

Reply:
Ah! I was following the below naming convention for the other TabletRecorder filter in the file here: https://github.com/vitessio/vitess/blob/master/go/vt/discovery/topology_watcher.go#L361

I am happy to change it though, if you prefer to stay consistent with the naming conventions for the filtering server.